docs/proposals/00-template.md

```diff
@@ -0,0 +1,20 @@
+# Headline of GEP
```

Comment:
Wouldn't this make more sense to be a pull request template?

Reply:
Hm what do you mean? I can add an additional PR template with necessary information about GEPs but if we put the GEP template into the PR template, we wouldn't have the GEP as a separate file any more, right?

Reply:
Yes, sorry, I meant to have an additional PR template for GEPs that points to this template and your other documentation.

Reply:
As discussed, I added a note to the general PR template.